<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RegexWidget</class>
 <widget class="QWidget" name="RegexWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>883</width>
    <height>696</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="locale">
   <locale language="English" country="UnitedStates"/>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLineEdit" name="lineEdit">
       <property name="placeholderText">
        <string>Regex</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="copyRegexButton">
       <property name="toolTip">
        <string>Copy</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset theme="QIcon::ThemeIcon::EditCopy"/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pasteRegexButton">
       <property name="toolTip">
        <string>Paste</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset theme="QIcon::ThemeIcon::EditPaste"/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="clearRegexButton">
       <property name="toolTip">
        <string>Clear</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset theme="QIcon::ThemeIcon::EditClear"/>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Orientation::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="errorLabel">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="openTextButton">
       <property name="toolTip">
        <string>Open</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset theme="QIcon::ThemeIcon::DocumentOpen"/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="copyTextButton">
       <property name="toolTip">
        <string>Copy</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset theme="QIcon::ThemeIcon::EditCopy"/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pasteTextButton">
       <property name="toolTip">
        <string>Paste</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset theme="QIcon::ThemeIcon::EditPaste"/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="clearTextButton">
       <property name="toolTip">
        <string>Clear</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset theme="QIcon::ThemeIcon::EditClear"/>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="CodeEditor" name="plainTextEdit">
     <property name="placeholderText">
      <string>Put text here</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>CodeEditor</class>
   <extends>QPlainTextEdit</extends>
   <header>editor/codeeditor.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
